tests/boulder-integration.sh

```diff
@@ -44,6 +44,22 @@ common --domains le3.wtf install \
        --cert-path "${root}/csr/cert.pem" \
        --key-path "${root}/csr/key.pem"
 
+# This won't renew (because it's not time yet)
+common renew
+
+# This will renew
+sed -i "4arenew_before_expiry = 10 years" "$root/conf/renewal/le1.wtf.conf"
+common renew
```

Comment:
We should actually verify that the original `common renew` call didn't renew the certificates and the second one did by examining the configuration directory. I'm fine with just making an issue about this for later though.